README.md

```diff
@@ -231,6 +275,8 @@ Apostrophe hosting will automatically handle:
 - Automatic backups
 - Security updates
 
+In the future, we will be providing a path to create your own account and create a new hosted project. In the meantime, you can [contact us](https://apostrophecms.com/contact-us) to get your hosting set-up.
```

Comment:
"set up" (because it's a verb in this case; the hyphen is not usual)

Reply:
Sorry I missed this in the last comment. Fixed.